### ISSUE TITLE
Add option to disable checkout descriptions

### ISSUE THREAD

**Issue by nabeghe** [easydigitaldownloads/easy-digital-downloads#6968]
Please add the 'File Downloads' settings in the Misc tab for each product Separately.

**Comment by nabeghe** [comment]
> 
> 
> @nabeghe Is there a specific setting you are wanting? Could you describe the business need for this a little further so we can better understand the requirements?

The 'Download settings' Box, just contains the 'File Download Limit' option! But I need to specify 'Download Link Expiration' and 'Disable Redownload' for each product separately.

![1](https://user-images.githubusercontent.com/12207627/46434167-b3f81280-c75f-11e8-9697-7f0966f58542.jpg)

**Comment by pippinsplugins** [comment]
Definitely a valid use case and makes sense though I'd like to hold off on this until we see more votes for it.

**Comment by cklosowski** [comment]
@pippinsplugins Yeah that is kind of my feeling on it. I'll leave it open for now.
